src/data_info/clients/async_tasks.clj

```diff
@@ -18,9 +20,24 @@
                      (handler e &throw-context)
                      {::exception e})))]
     (if (::exception res)
-      (recur (dec retries) handler f args)
+      (do
+        (Thread/sleep 2000) ;; arbitrary wait to allow for things to improve
+        (recur (dec retries) handler f args))
```

Comment:
Not requesting this change, but I just wanted to point out that this could also have some kind of back-off logic, as we do in the DOI request service:
https://github.com/cyverse-de/terrain/blob/e8187e1a3b1dd59c5e701c9e7e02dd7ed212b558/src/terrain/services/permanent_id_requests.clj#L200-L225